cortex_search_create.sql

```diff
@@ -0,0 +1,18 @@
+USE DATABASE cortex_analyst_demo;
+USE SCHEMA revenue_timeseries;
+
+CREATE OR REPLACE DYNAMIC TABLE product_landing_table
+  WAREHOUSE = cortex_analyst_wh
+  TARGET_LAG = '1 hour'
+  AS (
+      SELECT DISTINCT product_line AS product_dimension FROM product_dim
+  );
+
+
+  CREATE OR REPLACE CORTEX SEARCH SERVICE product_line_search_service
+  ON product_dimension
+  WAREHOUSE = cortex_analyst_wh
+  TARGET_LAG = '1 hour'
+  AS (
+      SELECT product_dimension FROM product_landing_table
+  );
```

Comment:
The cortex search team fixed the limitation where `distinct` was not working, we can remove the dynamic table creation and just create the services `as SELECT DISTINCT product_line AS product_dimension FROM product_dim`